Week3/homework/step2-3.js

```diff
@@ -18,8 +21,15 @@ function repeatStringNumTimesWithWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num <= 0) {
+    return result;
+  }
```

Comment:
This test is redundant and can be removed. The `while` loop condition is tested at the start of the loop, therefore the loop body will not be executed if `num <= 0`.